modules/gis/src/lib/binary-to-geojson.ts

```diff
@@ -72,7 +74,7 @@ export function getSingleFeature(data: BinaryFeatures, globalFeatureId: number)
     }
   }
 
-  return null;
+  throw new Error(`featureId:${globalFeatureId} not found`);
```

Comment:
Do we want this function to throw? Also overhead of error message...

In a typescript app, typing will alert the user to the fact that the function can return null...

Reply:
The only scenario that `binaryToGeoJson` could return null is if the `globalFeatureId` option is specified (single feature mode) and not found. If the return type could be null, then even if the user does not use the single feature mode, they still have to check the returned object for null, which adds unnecessary complexity to the application.

Reply:
Agreed. (Though I supposed that could have been a possible argument for providing separate functions).